mri/operators/fourier/utils.py

```diff
@@ -150,7 +150,8 @@ def get_stacks_fourier(kspace_loc, volume_shape):
 
     try:
         idx_mask_z = np.asarray([
-            np.where(x == full_stack_z_loc)[0][0] for x in sampled_stack_z_loc
+            np.where(np.isclose(x, full_stack_z_loc))[0][0]
+            for x in sampled_stack_z_loc
```

Comment:
we should change `x` to something more readable while we are at it. I was confused as to what it meant.